src/assets/stylesheets/_social_cards.scss

```diff
@@ -221,3 +230,31 @@ $blueDarken: #0085ac;
     min-height: 157px;
   }
 }
+
+
+// --------------------------------------------------------------- //
+// scrivito_dark adaptions
+// --------------------------------------------------------------- //
+
+.scrivito_dark {
+  background: #3f3f3f;
+
+  .scrivito_detail_label span {
+    color: #eee;
```

Comment:
* [ ] This overrides the expected color of the label if there are validation issues

![image](https://user-images.githubusercontent.com/986170/140192152-9c521538-fcac-454f-bd30-0ca88d551bc7.png)

"Creator" should be orange here.

Reply:
See https://github.com/Scrivito/scrivito_example_app_js/pull/428#discussion_r742639618